scripts/NCBI_Assembly/README.md

```diff
@@ -0,0 +1,94 @@
+
+### License
+
+This data is from an NIH National Library of Medicine (NLM) genome unrestricted-access data repository and made accessible under the [NIH Genomic Data Sharing (GDS) Policy](https://osp.od.nih.gov/scientific-sharing/genomic-data-sharing/) and the [NLM Accessibility policy](https://www.nlm.nih.gov/accessibility.html). Additional information on "NCBI Website and Data Usage Policies" can be found [here](https://www.ncbi.nlm.nih.gov/home/about/policies/).
+
+### Dataset Documentation and Relevant Links
+
+The database original documentation is accessible on [NCBI Assembly](https://www.ncbi.nlm.nih.gov/assembly/help/). NCBI assembly reports can be accessed using [FTP Download](https://ftp.ncbi.nlm.nih.gov/genomes/all/).
+
+## About the import
+
+### Artifacts
+
+#### Scripts
+
+`format_NCBI_Chromosome.py`
+
+#### Files
+
+`GCF_000001405.39_GRCh38.p13_assembly_report.txt`
+
+### Import Procedure
+
+#### Processing Steps 
+
+To generate the data mcf from assembly reports datasets from NCBI Assembly run:
+
+```bash
+python3 format_NCBI_Chromosome.py file_input file_output genome species_abrv
+```
+Description: Converts an NIH NCBI assembly reports file on a genome assembly into a mcf output file populating "GenomeAssembly", "GenomeAssemblyUnit", and "Chromosome" entities.
+
+@file_input		path to the input NCBI file on a genome assembly
+@file_output	path to the output mcf file to write the reformatted data
+@genome 		the shorthand name for the genome assembly represented in the file (e.g. hg38, mm10)
+@species_abrv	the abbreviation used by Data Commons to represent species dcid (e.g. hs, mm)
+
+#### Post-Processing Steps
+
+Review output file for issues in incorrect parsing of whitespace for examples of each entity generated "GenomeAssembly", "GenomeAssemblyUnit", and "Chromosome". If incorrect parsing occurs, update `format_NCBI_Chromosome.py` to ensure correct parsing of properties and thier associated values for each entity.
+
+We also recommend checking the input file for new properties or ennumeration types present in the input file. If this is the case, please update the schema GenomeAnnotationEnum.mcf for updates to ennumeration entities and GenomeAnnotation.mcf for updates in properties. Schema changes are manually curated and not script generated for this script. Please note that an error will be thrown for new properties of "GenomeAssembly", but not "GenomeAssemblyUnit" or "Chromosome". There will be a custom error message for new enummeration entities in the input file. Please update the list of current enummeration values for each Enummeration class in `format_NCBI_Chromosome.py` in addition to adding them to the schema to ensure that the message is only thrown when an enummeration type is not currently in the schema.
+
+### Test
+
+To practice generating a data mcf from an assembly report dataset, in this case hg38 (GRCh38) run:
```

Comment:
Its good practice to stash the expected MCF file to compare against the output.

So we would check-in a file like testdata/hg38_genome_assembly_expected.mcf and then in the instructions say, something like below...

To validate generating a data mcf from an assembly report dataset, in this case hg38 (GRCh38) run:

```
$ python format_NCBI_Chromosome.py GCF_000001405.39_GRCh38.p13_assembly_report.txt hg38_genome_assembly_actual.mcf hg38 hs
$ diff hg38_genome_assembly_actual.mcf testdata/hg38_genome_assembly_expected.mcf
```

This should not return any diffs.